pkg/helpers/klusterlet/klusterlet.go

```diff
@@ -0,0 +1,24 @@
+// Copyright Contributors to the Open Cluster Management project
+package klusterlet
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ghodss/yaml"
+	"k8s.io/klog/v2"
+	"open-cluster-management.io/ocm/pkg/operator/helpers/chart"
+)
+
+// MergeKlusterletValues merges a klusterlet values file into the klusterlet chart config.
+func MergeKlusterletValues(klusterletValuesFile string, klusterletChartConfig *chart.KlusterletChartConfig) error {
+	values, err := os.ReadFile(klusterletValuesFile)
+	if err != nil {
+		return fmt.Errorf("failed to read klusterlet values file %s: %v", klusterletValuesFile, err)
+	}
+	if err := yaml.Unmarshal(values, klusterletChartConfig); err != nil {
```

Comment:
would add somme comment on how this is merged if some fields of klusterletChartConfig is set.

Reply:
resolved in [9612f72](https://github.com/open-cluster-management-io/clusteradm/pull/506/commits/9612f725195e923617b3e75b51e27a3520c9acc8)